src/views/Swap/Swap.vue

```diff
@@ -429,6 +429,9 @@ export default {
     account () {
       return this.accountItem(this.fromAccountId)
     },
+    isHighFee () {
+      return this.toSwapFee >= this.sendAmount * 0.25
```

Comment:
why only `toSwapFee`? That is the fee paid only for the receiving side of the swap. `fromSwapFee` is another fee which will be paid as part of the swap.

you also can't compare sendAmount to `toSwapFee` because they will be amounts for 2 different assets.
for example 
1 BTC - 50 ETH

`toSwapFee` will be the fee paid to claim ETH
`sendAmount` is 1 BTC
so this comparison cannot be made

I think you need to compare something like the total fee in fiat vs the receive amount in fiat